actions/class.ItemImport.php

```diff
@@ -50,7 +50,10 @@ protected function getAvailableImportHandlers()
 
         foreach (array_keys($returnValue) as $key) {
             if ($returnValue[$key] instanceof \tao_models_classes_import_CsvImporter) {
-                $importer = new CsvImporter();
+                //@TODO @FIXME Inject this in the proper way
```

Comment:
```suggestion
```
Could it be dropped from this comment or inject in a proper way in scope of this PR?

Reply:
Yes. this will be fixed. But I do not want to do it before get reviews to avoid rework :) 

Thanks

Reply:
Done